relay-server/src/actors/processor.rs

```diff
@@ -1079,12 +1079,22 @@ impl EnvelopeProcessorService {
     }
 
     /// Remove replays if the feature flag is not enabled
-    fn process_replays(&self, state: &mut ProcessEnvelopeState) {
+    fn process_replays(&self, state: &mut ProcessEnvelopeState) -> Result<(), ProcessingError> {
         let replays_enabled = state.project_state.has_feature(Feature::Replays);
         let context = &state.envelope_context;
         let envelope = &mut state.envelope;
         let client_addr = envelope.meta().client_addr();
 
+        // TODO: single function combines pii configs into one.
+        let pii_config1 = state.project_state.config().pii_config.as_ref();
+        let pii_config2 = state
+            .project_state
+            .config()
+            .datascrubbing_settings
+            .pii_config()
+            .map_err(|e| ProcessingError::PiiConfigError(e.clone()))?
+            .as_ref();
```

Comment:
Advice on potential configuration is needed here.

Reply:
This would give you some sensible defaults

```rust
DataScrubbingConfig {
    scrub_data: true,
    scrub_defaults: true,
    ..Default::default()
}.pii_config()
```

equivalent to

![image](https://user-images.githubusercontent.com/1565449/199995388-b4816e2a-b4a5-4eb4-9c7e-8f163d77a0e3.png)

Reply:
I didn't use `..Default::default()`.  I was getting an error that `pii_config` is private.  So I used:

```rust
let mut scrub_config = DataScrubbingConfig::default();
scrub_config.scrub_data = true;
scrub_config.scrub_defaults = true;
```

Is this equivalent?

Reply:
My bad, didn't think of that! Yes, it's equivalent.